source/_integrations/uptime_kuma.markdown

```diff
@@ -0,0 +1,20 @@
+---
+title: Uptime Kuma
+description: Instructions on how to integrate Upttime Kuma monitors into Home Assistant.
+ha_category:
+  - Binary Sensor
+ha_iot_class: Cloud Polling
+ha_release: 2021.12
+ha_domain: uptime_kuma
+ha_config_flow: true
+ha_codeowners:
+  - '@meichthys'
+ha_platforms:
+  - binary_sensor
+---
+
+The Uptime Kuma integration exposes the Uptime Kuma `/metrics` endpoint information in Home Assistant.
```

Comment:
Add a description on what Uptime Kuma is.

I'm not sure if telling `/metrics` endpoint is used, is something the end-user is interested in / needs to know.

Maybe add a link to Uptime Kuma itself as well?